.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,39 +1,35 @@
 <!---
+
 Thanks for creating a pull request!
 
-Please start your PR title with the appropriate prefix as specified by [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#specification):
+Checklist:
 
-chore: [your title]
-docs: [your title]
-feat: [your title]
-fix: [your title]
-refactor: [your title]
-rfc: [your title]
-test: [your title]
--->
+ - Documentation in README, crate, module, function and/or other locations added/adapted.
 
+ - New tests added and/or existing tests adapted.
 
-## Intent
-<!---
-Please describe what you want to achieve here.
-Keep your PR limited to one logical change.
-For example, if you add a feature, don't include fixes you made along the way
-Please open another PR for more contributions.
--->
+ - PR is limited to a single logical change.
+   For example, if you add a feature, don't include fixes you made along the way
```

Comment:
> Is this something we decided on? I remember the team having split opinions on this

Yes, I think there was split opinion, but the template wasn't updated, this is just moving the existing checklist around. If someone remembers what the sort of consensus around it was it would be good to change it in another PR (I don't remember if it was to outright remove this, or just to weaken it in some way).